platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateSentinel.java

```diff
@@ -25,7 +26,9 @@
 public interface SignedStateSentinel {
     /**
      * Check the maximum age of signed states, and take action if a really old state is observed.
+     *
+     * @param now the current time
      */
     @InputWireLabel("heartbeat")
-    void checkSignedStates();
+    void checkSignedStates(final Instant now);
```

Comment:
```suggestion
    void checkSignedStates(@NonNull final Instant now);
```

Reply:
added in my repo.   I didn't adopt the suggestion because the IDE needs to add the dependency for the annotation to be used.